### ISSUE TITLE
Raise issues corresponding to open, rejected and reversed issues / pull requests from legacy project

### ISSUE THREAD

**Issue by ajburley** [hawknetwork/sealnote-plus#29]
Raise issues on this project corresponding to the open issues and pull requests on the legacy SealNote project. Also consider rejected issues and PRs from legacy, such as 36, and any closed issue or PR that had its result subsequently reversed either in this project or legacy, and raise issues if necessary. Go through these and process them:

- Close as appropriate (e.g. won't fix, obsolete, already implemented, duplicate etc)
- Mark up with suitable labels and provide explanations
- Assign to milestone(s) as appropriate. In particular any issue assigned to the Release 1.0 project in SealNote legacy should be strongly considered for assignment to a milestone in SealNote Plus
- Where multiple items are raised in a single issue, create multiple additional issues and then close the original one

Also, note any that are not being raised (from closed issues/PRs) in the comments on this issue.

This will enable us to continue working on those issues that are still relevant.

**Comment by ajburley** [comment]
I would like to do some basic tasks here before starting to link with legacy (which may trigger notifications etc). In particular I would like to see #17, #19, #20, #24, #35, #36, #37 and #56 resolved first.

It would also be nice to resolve or at least make progress on #18 and #31.

**Comment by ajburley** [comment]
Also consider closed issue 36 from legacy.

**Comment by ajburley** [comment]
If everything else for Release 0.9 is complete, it would be good to have a patch in hand for the Android 8+ backup issue, so that I could raise -> patch/close -> tag -> publish in one go.
